bundles/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/handler/ADBridgeHandler.java

```diff
@@ -354,7 +355,8 @@ private void notifyChildHandlersPanelReady() {
             // Notify child zone handlers by calling notifyPanelReady() for each
             for (Thing thing : getThing().getThings()) {
                 ADThingHandler handler = (ADThingHandler) thing.getHandler();
-                if (handler != null && (handler instanceof ZoneHandler || handler instanceof RFZoneHandler)) {
+                if (handler != null && (handler instanceof ZoneHandler || handler instanceof RFZoneHandler
+                        || handler instanceof VZoneHandler)) {
                     handler.notifyPanelReady();
```

Comment:
This would be more maintainable if all of the zone handlers shared the same parent class or implemented the same interface.

It also might just be simpler to just call `notifyPanelReady()` on all handlers since only the zone handlers have a non-empty implementations anyway.

Reply:
>It also might just be simpler to just call notifyPanelReady() on all handlers since only the zone handlers have a non-empty implementations anyway.

That's a good point. I've updated notifyChildHandlersPanelReady() in ADBridgeHandler to call notifyPanelReady() for all child things.

>This would be more maintainable if all of the zone handlers shared the same parent class or implemented the same interface.

They all extend ADThingHandler, which specifies most of the interface. The various zone handlers don't have much more in common with each other than they do with the other child thing handlers, so I don't think too much more could be abstracted out.